AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -153,27 +152,24 @@ public IActionResult Register()
         // GET: /Account/ConfirmEmail
         [HttpGet]
         [AllowAnonymous]
-        public async Task<IActionResult> ConfirmEmail(string userId, string code)
+        public async Task<IActionResult> ConfirmEmail(string userId, string token)
         {
-            if (userId == null || code == null)
-            {
+            if (userId == null || token == null)
                 return View("Error");
-            }
+

```

Comment:
We should add this to the style guide, but we do prefer having the braces around if and else clauses, even for single statement clauses.